README.md

```diff
@@ -199,3 +199,7 @@ These are not meant to be comprehensive _by any means_. You may even find confli
 ## Other Non-Technical Topics
 
 - [Project handoff guide by Simon](soapbox/general/smonn-project-handoff.md)
+
+## Python
```

Comment:
Please add an anchor on the table of contents for this section if you would!

Reply:
I also added an anchor for angular